<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LeastSquare</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="InputTable" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>567</x>
      <y>10</y>
      <width>180</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>663</x>
      <y>320</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="Figure" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>551</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>320</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="currentText">
     <string>3</string>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>320</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>order</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>InputTable</class>
   <extends>QTableWidget</extends>
   <header>inputtable.h</header>
  </customwidget>
  <customwidget>
   <class>Figure</class>
   <extends>QWidget</extends>
   <header>figure.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>dataInput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>685</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>dataInput(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>633</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>dataInput()</slot>
  <slot>dataInput(QString)</slot>
 </slots>
</ui>
